Makefile

```diff
@@ -47,20 +48,20 @@ cleandocker:
 	# Stop all containers (if running)
 	docker-compose -f docker/docker-compose.yml stop
 	# Remove mainflux containers
-	docker ps -f name=mainflux -aq | xargs -r docker rm
+	docker ps -f name=$(MF_DOCKER_IMAGE_NAME_PREFIX) -aq | xargs -r docker rm
 
 	# Remove exited containers
-	docker ps -f name=mainflux -f status=dead -f status=exited -aq | xargs -r docker rm -v
+	docker ps -f name=$(MF_DOCKER_IMAGE_NAME_PREFIX) -f status=dead -f status=exited -aq | xargs -r docker rm -v
 
 	# Remove unused images
-	docker images "mainflux\/*" -f dangling=true -q | xargs -r docker rmi
+	docker images "$(MF_DOCKER_IMAGE_NAME_PREFIX)\/*" -f dangling=true -q | xargs -r docker rmi
```

Comment:
Are the quotes really needed in `"$(MF_DOCKER_IMAGE_NAME_PREFIX)\/*"` or can be removed?

Reply:
I think it depends on that is `sh` on your system